frontend/server/src/DAO/Problems.php

```diff
@@ -1157,4 +1157,18 @@ public static function hasBeenUsedInCoursesOrContests(
         /** @var list<array{name: string, problems_per_tag: int}> */
         return \OmegaUp\MySQLConnection::getInstance()->GetAll($sql);
     }
+
+    final public static function getProblemAliasRandom(): string {
```

Comment:
nit:

```suggestion
    final public static function getRandomProblemAlias(): string {
```

aquí "random" es un adjetivo que describe el "ProblemAlias" (o más correctamente, al "alias"). En inglés, típicamente se ponen todos los adjetivos antes del sustantivo al que se refieren.